[2024-05-22 02:20:43,223] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 02:20:43,346] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 02:20:43,348] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 02:20:43,349] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 02:20:43,351] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 02:20:43,458] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-22 01:00:00+00:00
[2024-05-22 02:20:43,493] {standard_task_runner.py:52} INFO - Started process 2409 to run task
[2024-05-22 02:20:43,583] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-22T01:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmp59rejh5t', '--error-file', '/tmp/tmpaxz7c2wa']
[2024-05-22 02:20:43,631] {standard_task_runner.py:77} INFO - Job 27: Subtask fetch_data
[2024-05-22 02:20:44,246] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [running]> on host 8fe6997c1526
[2024-05-22 02:20:44,612] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-22T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-22T01:00:00+00:00
[2024-05-22 02:20:44,794] {logging_mixin.py:109} INFO - Failed to retrieve data: 403
[2024-05-22 02:20:44,800] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 02:20:44,939] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240522T010000, start_date=20240522T022043, end_date=20240522T022044
[2024-05-22 02:20:45,032] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 02:20:45,256] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22 04:17:56,756] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 04:17:56,826] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 04:17:56,826] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 04:17:56,826] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 04:17:56,827] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 04:17:56,891] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-22 01:00:00+00:00
[2024-05-22 04:17:56,902] {standard_task_runner.py:52} INFO - Started process 392 to run task
[2024-05-22 04:17:56,917] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-22T01:00:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmp60_krhze', '--error-file', '/tmp/tmpp2p1f86f']
[2024-05-22 04:17:56,950] {standard_task_runner.py:77} INFO - Job 47: Subtask fetch_data
[2024-05-22 04:17:57,084] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [running]> on host 3419eee920a5
[2024-05-22 04:17:57,294] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-22T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-22T01:00:00+00:00
[2024-05-22 04:17:57,302] {logging_mixin.py:109} INFO -  api is None
[2024-05-22 04:17:57,308] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Clast_seen&location_type=0&max_age=300
[2024-05-22 04:17:57,309] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': 'None'}
[2024-05-22 04:17:57,439] {logging_mixin.py:109} INFO - Status Code: 403
[2024-05-22 04:17:57,440] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716351477,
  "error" : "ApiKeyInvalidError",
  "description" : "The provided api_key was not valid."
}
[2024-05-22 04:17:57,510] {logging_mixin.py:109} INFO - Failed to retrieve data: 403
[2024-05-22 04:17:57,511] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 04:17:57,554] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240522T010000, start_date=20240522T041756, end_date=20240522T041757
[2024-05-22 04:17:57,656] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 04:17:57,744] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-22 06:56:36,958] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 06:56:37,101] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 06:56:37,102] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 06:56:37,102] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 06:56:37,102] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 06:56:37,240] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-22 01:00:00+00:00
[2024-05-22 06:56:37,307] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-22T01:00:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpzfm4hvw5', '--error-file', '/tmp/tmp93qxt21b']
[2024-05-22 06:56:37,388] {standard_task_runner.py:77} INFO - Job 64: Subtask fetch_data
[2024-05-22 06:56:37,281] {standard_task_runner.py:52} INFO - Started process 585 to run task
[2024-05-22 06:56:37,872] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 06:56:38,318] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-22T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-22T01:00:00+00:00
[2024-05-22 06:56:38,327] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 06:56:38,327] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 06:56:38,479] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 06:56:38,480] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716360998,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 06:56:49,365] {logging_mixin.py:109} INFO - Failed to retrieve data: 500
[2024-05-22 06:56:49,372] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 06:56:50,271] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240522T010000, start_date=20240522T065636, end_date=20240522T065650
[2024-05-22 06:56:50,518] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 06:56:50,832] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-22 07:05:01,725] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 07:05:01,910] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 07:05:01,911] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:05:01,911] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 07:05:01,911] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:05:02,107] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-22 01:00:00+00:00
[2024-05-22 07:05:02,158] {standard_task_runner.py:52} INFO - Started process 1142 to run task
[2024-05-22 07:05:02,200] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-22T01:00:00+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpgrodokrp', '--error-file', '/tmp/tmpe0pf8hln']
[2024-05-22 07:05:02,273] {standard_task_runner.py:77} INFO - Job 141: Subtask fetch_data
[2024-05-22 07:05:02,633] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 07:05:03,186] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-22T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-22T01:00:00+00:00
[2024-05-22 07:05:03,313] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 07:05:03,313] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 07:05:03,884] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 07:05:03,884] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716361503,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 07:05:04,180] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 07:05:04,186] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 07:05:04,376] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240522T010000, start_date=20240522T070501, end_date=20240522T070504
[2024-05-22 07:05:04,837] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 07:05:05,180] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-22 07:12:55,173] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 07:12:55,341] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 07:12:55,350] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:12:55,351] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 07:12:55,364] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:12:55,587] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-22 01:00:00+00:00
[2024-05-22 07:12:55,673] {standard_task_runner.py:52} INFO - Started process 1680 to run task
[2024-05-22 07:12:55,835] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-22T01:00:00+00:00', '--job-id', '223', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpfk27ppw_', '--error-file', '/tmp/tmpluvyci0y']
[2024-05-22 07:12:55,964] {standard_task_runner.py:77} INFO - Job 223: Subtask fetch_data
[2024-05-22 07:12:56,824] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 07:12:57,329] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-22T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-22T01:00:00+00:00
[2024-05-22 07:12:57,380] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 07:12:57,384] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 07:12:57,661] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 07:12:57,662] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716361977,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 07:12:57,961] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 07:12:57,962] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 07:12:58,068] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240522T010000, start_date=20240522T071255, end_date=20240522T071258
[2024-05-22 07:12:58,282] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 07:12:58,503] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-22 07:22:34,182] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 07:22:34,305] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [queued]>
[2024-05-22 07:22:34,305] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:22:34,306] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2024-05-22 07:22:34,306] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2024-05-22 07:22:34,405] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): fetch_data> on 2024-05-22 01:00:00+00:00
[2024-05-22 07:22:34,467] {standard_task_runner.py:52} INFO - Started process 2364 to run task
[2024-05-22 07:22:34,523] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'purpleair_api_dag', 'fetch_data', 'scheduled__2024-05-22T01:00:00+00:00', '--job-id', '333', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpgwn5pi52', '--error-file', '/tmp/tmpob1yn_oz']
[2024-05-22 07:22:34,594] {standard_task_runner.py:77} INFO - Job 333: Subtask fetch_data
[2024-05-22 07:22:35,017] {logging_mixin.py:109} INFO - Running <TaskInstance: purpleair_api_dag.fetch_data scheduled__2024-05-22T01:00:00+00:00 [running]> on host 148b4063f71c
[2024-05-22 07:22:35,384] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=purpleair_api_dag
AIRFLOW_CTX_TASK_ID=fetch_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-22T01:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-22T01:00:00+00:00
[2024-05-22 07:22:35,405] {logging_mixin.py:109} INFO - Full URL: https://api.purpleair.com/v1/sensors?fields=latitude%2Clongitude%2Chumidity%2Cpm2.5_60minute%2Ctemperature&location_type=0&max_age=300
[2024-05-22 07:22:35,406] {logging_mixin.py:109} INFO - Headers: {'X-API-Key': '9A23CD90-1279-11EF-B9F7-42010A80000D'}
[2024-05-22 07:22:35,746] {logging_mixin.py:109} INFO - Status Code: 402
[2024-05-22 07:22:35,747] {logging_mixin.py:109} INFO - Response Content: {
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716362555,
  "error" : "RateLimitExceededError",
  "description" : "Available API call limit exceeded, please try later."
}{
  "api_version" : "V1.0.13-0.0.56",
  "time_stamp" : 1716362555,
  "error" : "PaymentRequiredError",
  "description" : "Payment is required to make this api call."
}
[2024-05-22 07:22:35,939] {logging_mixin.py:109} INFO - Failed to retrieve data: 402
[2024-05-22 07:22:35,962] {python.py:175} INFO - Done. Returned value was: None
[2024-05-22 07:22:36,053] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=purpleair_api_dag, task_id=fetch_data, execution_date=20240522T010000, start_date=20240522T072234, end_date=20240522T072236
[2024-05-22 07:22:36,430] {local_task_job.py:154} INFO - Task exited with return code 0
[2024-05-22 07:22:36,949] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
